Sources/HTTPParser/http_parser.swift

```diff
@@ -176,7 +312,7 @@ private let F_SKIPBODY: http_flags              = 1 << 6
 private let F_CONTENTLENGTH: http_flags         = 1 << 7
 
 /* Map for errno-related constants */
-public enum http_errno: String, Error {
+public enum http_errno: StaticString, Error {
```

Comment:
Actually I was kinda wrong on this one. I was assuming that a Swift String `enum` would use Strings at runtime to match cases. That actually doesn't seem to be the case. The enum is always some small integer and it the compiler generates the switch to do the raw value mapping from/to string.

But for the specific function you mention, that kinda works for any enum: "\(err)" and matches what the C variant does.